README.md

```diff
@@ -26,7 +26,15 @@ Link to portfolio:
 - Open your Gitbash or your code editor's terminal and run the command git clone https://github.com/Bantealem/Portfolio-setup-and-mobile-version-skeleton
 - Then you can pull the work from github to Your local machine using the git pull command.
 
+## Deployment
 
+To deploy this project run
+
+```bash
+  npm run deploy
+```
+
+https://bantealem.github.io/Portfolio-setup-and-mobile-version-skeleton/
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you make your link clickable by formating it:
## EXAMPLE
```
## Live Demo
`[LIve Demo](https://bantealem.github.io/Portfolio-setup-and-mobile-version-skeleton/)`
```